lib/galaxy/util/checkers.py

```diff
@@ -49,9 +50,14 @@ def check_binary(name, file_path=True):
     # Handles files if file_path is True or text if file_path is False
     if file_path:
         temp = open(name, "rb")
+        read_start = int(os.stat(name).st_size/2)
```

Comment:
```suggestion
        read_start = int(os.stat(name).st_size / 2)
```

Reply:
Thanks, sorry, I had added the fix before seeing this, otherwise I would have committed your suggestion.